charter.md

```diff
@@ -53,15 +53,164 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+few methods:
+
+* Dissolution by vote
+* Dissolution by loss of committee
+
+See [elections](#elections) for special criteria in the instance of a general
+election triggered by dissolution.
+
+#### Dissolution by vote
+
+The committee must specify a date of dissolution. Dissolution requires an
+affirmative vote of 75% of committee members, and must be unanimous when the
+committee has four (4) or fewer members. If the committee is dissolved, all
+seats are vacated on the date specified.
+
+#### Dissolution by loss of committee
+
+In the event that 50% (or more) of the committee's members
+[resign](#resignation) at the same time, the entire committee will be disbanded.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by a three-fourths vote of no
+confidence from other Steering Committee members.
+
+The call for a vote of no confidence will happen in a public Steering Committee meeting and a
+GitHub issue in kubernetes/steering by a current member and must be seconded by
+another current member.
+
+There will be a feedback period of one week for members of the community to reach
+out to Steering Committee members with their thoughts or log them on the open
+issue, including non-binding votes. After this feedback period, Steering Committee
+members must vote on the issue within 48 hours.
+
+## Voting
+
+In the course of the committee's operations, members will be expected to vote
+on decisions within the body's purview.
+
+These votes may proceed on agreed-upon platforms by the committee, such as:
+
+* a pull request
+* an issue
+* a Steering Committee [meeting](#meetings)
+* a mailing list
+
+Unless otherwise specified by a process, the requirement for passing a vote is
+a **_two-thirds supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* 5 members in attendance would be required to vote during a meeting
+
+### Abstention
+
+For any self-elected reason, members of the committee may decide to abstain
+from a vote.
+
+Abstaining members will only be considered as contributing to quorum, in the
+event that a vote is called in a meeting.
+
+## Meetings
+
+Steering Committee members are generally expected to attend every meeting. We
 use the following guidelines to determine whether we have reached quorum and
 are able to proceed with a meeting.
 
-* Round quorum up if total number of members is odd
-* Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
-* Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
+### Quorum
+
+Quorum **to meet** is a **_majority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* 4 members in attendance would be required to meet
+
+Quorum **to vote in a meeting** is a **_two-thirds supermajority of the
```

Comment:
This says that you need 50% to have a meeting, but 67% to vote in a meeting.

What's the point of a meeting where no votes can be taken? Why even have that in the rules?

Reply:
It's to prevent part of the board plotting against the rest of the board :/

At least that was my understanding of similar provisions I've seen.

Reply:
> What's the point of a meeting where no votes can be taken? Why even have that in the rules?

There are meetings where we do not have votes on the agenda, so in many instances, a quorum of a majority of the fixed membership of the committee is sufficient to proceed.